examples/dodge-the-creeps/rust/src/player.rs

```diff
@@ -1,4 +1,6 @@
-use godot::engine::{AnimatedSprite2D, Area2D, Area2DVirtual, CollisionShape2D, PhysicsBody2D};
+use godot::engine::{
+    AnimatedSprite2D, Area2D, Area2DVirtual, CollisionShape2D, Engine, PhysicsBody2D,
+};
```

Comment:
Interesting that this changed -- is rustfmt behaving different in recent Rust versions?

Reply:
I don't use rustfmt much; does it care about how long lines get? That'd prob be it.

Reply:
it's because `Engine` was added

Reply:
Time to go to sleep 🙈